content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+He confirmed that what Bartlett called "representative detached details", that is parts of a picture that represent a distinctive pattern or object even when isolated from the whole, are well preserved both in artificial and historical serial reproductions.
+Ward thus suggested that comparing historical data to results of serial reproduction experiments can be a useful method to investigate the influence of universal-psychological versus local-cultural factors in theories of historical evolution of artefacts.
+@hall_effect_1950 further explored hypotheses made by Bartlett on the effect of titles for pictorial and verbal serial reproduction:
+in those new experiments, it appeared that titles had a considerable influence on the reproduction of both pictures and texts, by acting as an interpretive frame that guides (or confuses) the participants in interpreting the material.^[
+  Hall, noting that "the function of the headline or title to some story or article is that of giving a particular emphasis to certain aspects of the text, and is one of the main methods of distorting and biasing what is remembered" [@hall_effect_1950, p. 120], later reflects on the fact that such results have crucial political implications given the development of mass media.
+]
+More recently the reliability of Bartlett's studies has been discussed, shifting the focus towards a better control of experimental conditions:
+@gauld_experiments_1967, while still praising Bartlett for the progress that his work represented compared to that of his predecessors, showed that changing the exposition instructions, for instance by giving strict memorisation instructions, or by adding a simple sentence asking participants to be "accurate", would considerably reduce the transformation rate in the resulting chains, a fact that should question the effects measured by a setup that lacks explicit memorisation instructions.
+The authors also examined the level of conscientiousness of participants in accomplishing their task, showing that the measure correlated negatively to the participants' transformation rate.
+Noting that the effect was not explained away by a measure of intelligence of participants, they finally suggested that "errors could, it seemed, be avoided, if the subject was so inclined" [-@gauld_experiments_1967, p. 45].
+Today, one overcomes such an issue by designing experiments that create an intrinsic motivation for participants to adopt the behaviour under study [see @claidiere_argumentation_2017 for an example].
+In contemporary work, @kashima_recovering_2000 offered a reappraisal of the social aspect of Bartlett's contributions.
+Kashima argues that while Bartlett's legacy is mostly seen in psychological studies that adopt methodological individualism, Bartlett fiercely opposed that approach;
```

Comment:
TODO
- [ ] develop "psychological studies that adopt methodological individualism"
- [ ] develop what Bartlett has in common with 4E or embodied culture